content/show/this-week-in-bitcoin/0028.md

```diff
@@ -149,7 +149,7 @@
     ],
     "timesplits": [
       {
-        "startTime": 2844,
+        "startTime": 2755,
```

Comment:
Looks like this may no longer match the chapters.json linked by the feed?

[twib feed](https://serve.podhome.fm/rss/55b53584-4219-4fb0-b916-075ce23f714e)
[twib 28 chapters json](https://assets.podhome.fm/f01a19c0-6f9d-4aef-9515-08dc15242149/638628120200711135ca65d865-d8ff-4d3a-8789-f6e1ad8d68b6v1.chapters.json)

```json
    {
      "title": "Disco Swag",
      "url": "",
      "img": "https://www.doerfelverse.com/art/disco-swag.jpeg",
      "startTime": 2844.0
    }
```